.github/workflows/ci.yml

```
@@ -184,3 +184,9 @@ jobs:
       run: |
         cp .github/workflows/temp_centos_configure.sh temp_centos_configure.sh
         bash temp_centos_configure.sh
+
+    - name: Running small tests
+      run: |
+        export PYTHONPATH=${PYTHONPATH}:${GITHUB_WORKSPACE}/bin/Release
+        export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${GITHUB_WORKSPACE}/bin/Release/libs
+        python3 kratos/python_scripts/run_tests.py -l small -c python3
```

```suggestion
        python3.5 kratos/python_scripts/run_tests.py -l small -c python3.5
```

from experience :D
=> https://github.com/KratosMultiphysics/CoSimIO/pull/151/commits/01596d60a6d6f4a5070d4a99b2d7da77d20297c5

---

thanks ;)